### PR TITLE
Add misp-ioc-sender test util

### PR DESCRIPTION
**Pull request by 0ortmann**
###  :notebook_with_decorative_cover: Description

Add a testing script to inject well-formed `DOMAIN`-type indicators in MISP. The script will create all required resources, if necessary.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/threatbus](https://docs.tenzir.com/threatbus), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

- Setup a local MISP instance, e.g., via Docker. You can follow [these instructions](https://github.com/tenzir/threatbus/tree/master/plugins/apps/threatbus_misp#dockerized-misp) in the README of the `threatbus-misp` plugin.
- `cp config.yaml.example config.yaml`
- Place your MISP API key in `config.yaml`
- `pip install -r requirements.txt` (possibly use a virtualenv)
- `python misp-ioc-sender -c config.yaml`

The script logs what it does. You should verify in MISP that an event and attribute have been created. Play around with it, delete or rename resources in MISP, rerun the script. Try to break it.

You can also start Threat Bus with the MISP plugin loaded + some backbone. Verify that the backbone sees the IoC updates when you run this script.

### CODE REVIEW

File: tests/utils/misp-ioc-sender/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# MISP IoC Sender
+
+A script to trigger an Attribute update in MISP. The script is configured via a
+YAML file. See belog for details and check out the `config.yaml.example` file.
```
Comment (mavam) [inline tests/utils/misp-ioc-sender/README.md:4]:
```suggestion
YAML file. See below for details and check out the `config.yaml.example` file.
```

File: tests/utils/misp-ioc-sender/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+The script works as follows:
+
+- It looks for a specific MISP Event, referenced by its UUID in the config file.
+- The script creates the Event if it doesn't exist yet.
+- The script checks the Attributes of the Event and either creates a new
+  Attribute or it toggles the `'to_ids'` flag if the Attribute already exists.
```
Comment (mavam) [inline tests/utils/misp-ioc-sender/README.md:11]:
```suggestion
- Look for a specific MISP Event, referenced by its UUID in the config file.
- Create the Event if it doesn't exist.
- Check the Attributes of the Event and either create a new
  Attribute or it toggle the `'to_ids'` flag if the Attribute already exists.
```

File: tests/utils/misp-ioc-sender/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+Attributes created by this script always have the `"DOMAIN"` type. Attribute
```
Comment (mavam) [inline tests/utils/misp-ioc-sender/README.md:13]: Why always `DOMAIN`?

**Review by mavam — APPROVED** [review]
Only some minor text smithing from my side.

I don't have MISP instance handy, so couldn't do an E2E test. I read the code and couldn't find any obvious algorithmical flaws.

File: tests/utils/misp-ioc-sender/README.md (hunk 3 above)
Comment (0ortmann) [reply tests/utils/misp-ioc-sender/README.md:13]: That's what we agreed to use with DCSO for testing. We can make this configurable, if you'd like.

Comment (mavam) [reply tests/utils/misp-ioc-sender/README.md:13]: Let's keep it as is and update only if we need the extra flexibility.

**Pull request merged**